src/chem/aeronet_aaod2ioda.py

```diff
@@ -0,0 +1,221 @@
+#        downloaded from from NASA websit
+#        (https://aeronet.gsfc.nasa.gov/print_web_data_help_v3_inv_new.html)
+#        and converts to IODA format.
+#        Inversion products include conicident AOT data with almucantar
+#        retrieval (CAD), single scattering albedo (SSA),
+#        AOD aborption (TAB) for inversion types of ALM15 or ALM20 and
+#        at wavelengths of 440/675/870/1020nm)
+#
+# Usage:
+#        python aeronet_aaod2ioda.py -i testinput -o aeronet_aaod.nc
+#        -i: input path of AERONET inversion files
+#            (aeronet_cad.dat, aeronet_ssa.dat, aeronet_tab.dat)
+#        -o: output IODA file
+#
+# Contact:
+#        Bo Huang (bo.huang@noaa.gov) from CU/CIRES and NOAA/ESRL/GSL
+#        (September 28, 2021)
+#
+# Acknowledgement:
+#        Barry Baker from ARL for his initial preparation for this code.
+
+
+import netCDF4 as nc
+import numpy as np
+import inspect, sys, os, argparse
+import pandas as pd
+from datetime import datetime, timedelta
+from builtins import object, str
+from numpy import NaN
+from pathlib import Path
+
+
+IODA_CONV_PATH = Path(__file__).parent/"@SCRIPT_LIB_PATH@"
+if not IODA_CONV_PATH.is_dir():
+    IODA_CONV_PATH = Path(__file__).parent/'..'/'lib-python'
+sys.path.append(str(IODA_CONV_PATH.resolve()))
+import meteo_utils
+import ioda_conv_ncio as iconv
+from collections import defaultdict, OrderedDict
+from orddicts import DefaultOrderedDict
+
+
+def dateparse(x):
+    return datetime.strptime(x, '%d:%m:%Y %H:%M:%S')
+
+
+def add_data(infile):
+    df = pd.read_csv(infile,
+                     engine='python',
+                     header=None,
+                     skiprows=7,
+                     parse_dates={'time': [1, 2]},
+                     date_parser=dateparse,
+                     na_values=-999)
+    header = pd.read_csv(infile, skiprows=6, header=None,
+                         nrows=1).values.flatten()
+
+    cols = ['time']
+    for i in header:
+        if "Date(" in i or 'Time(' in i:
+            if "Last_Processing_Date(" in i or "Last_Processing_Time(" in i:
+                cols.append(i.lower())
+            else:
+                pass
+        else:
+            cols.append(i.lower())
+    df.columns = cols
+    df.index = df.time
+    df.rename(columns={
+        'latitude(degrees)': 'latitude',
+        'longitude(degrees)': 'longitude',
+        'elevation(m)': 'elevation',
+        'aeronet_site': 'siteid'
+    },
+        inplace=True)
+    return df
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description=(
+                     'Reads AERONET inversion files downloaded from NASA website '
+                     ' and converts into IODA format')
+    )
+
+    required = parser.add_argument_group(title='required arguments')
+    required.add_argument(
+                    '-i', '--input',
+                    help="path of AERONET inversion ASCII files that include "
```

Comment:
Make this explicitly say 'path to directory containing...' or something like that as this reads to me like you should provide it 3 paths `/path/to/aeronet_cad.dat /path/to/aeronet_ssa.dat` etc.